.circleci/config.yml

```diff
@@ -251,62 +250,17 @@ jobs:
           background: true
           command: cloud-provider-kind
       - run:
-          name: Cluster East
-          command: |
-            set -x
-            KUBECONFIG=~/kc.east ./scripts/kind-dev-cluster -n east \
-              --images ociarchive
-      - run:
-          name: Cluster West
+          name: Cluster E2E
           command: |
             set -x
-            KUBECONFIG=~/kc.west ./scripts/kind-dev-cluster -n west  \
+            KUBECONFIG=~/.kube/config ./scripts/kind-dev-cluster -n e2e-tests \
               --images ociarchive
       - run:
-          name: Initalize Sites
```

Comment:
I do not think this existing step should be replaced.
Differently than the E2E tests, this is validating example resources available
through `./cmd/controller/example`, which seems important to me as people
might use them, so it is important to make sure they are working as expected.

IMHO, the E2E should be a separate job that runs after this existing one.

Thoughts?

Reply:
I had steered @rafaelvzago towards dropping these, but that is an interesting point: testing ./cmd/controller/example. When I wrote this I had in my mind that this would be replaced w/ the e2e tests, so I mostly just hacked it together.

Do you think that we could find a way to work in exercising the ./cmd/controller/example in the tests/e2e scenarios? Or maybe it's nice to have a more shellscript-y scenario like this that uses the skupper cli build and stuff?